rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -271,7 +271,7 @@ public static int getServerSocketTimeout(Configuration conf)
     return conf.getInt(KEY_SERVER_SOCKET_TIMEOUT, DEFAULT_SERVER_SOCKET_TIMEOUT);
   }
 
-  public static int getPoolSizeMax(Configuration conf)
+  public static int getTranportPoolMaxSize(Configuration conf)
```

Comment:
sorry, I forgot to mention: getPoolSizeMin, getPoolDeltaSize, getPoolMaxWait

Reply:
done